File: Custom_Chat_App/Python/Bio.py
import os
from delphifmx import *
# Import chat form
from Chat import ChatScreen


class BioScreen(Form):

    def __init__(self, owner):
        self.TitleLabel = None
        self.AgeLabel = None
        self.NameLabel = None
        self.AgeNumberBox = None
        self.NameEdit = None
        self.Image1 = None
        self.StartButton = None
        self.StyleBook1 = None
        self.LoadProps(os.path.join(os.path.dirname(
            os.path.abspath(__file__)), "Bio.pyfmx"))

    def StartButtonClick(self, Sender):
        chat = ChatScreen(self, name=self.NameEdit.Text,
                          age=self.AgeNumberBox.Text)
        chat.Show()

File: Custom_Chat_App/Delphi/Bio.pas
unit Bio;

interface

uses
  System.SysUtils, System.Types, System.UITypes, System.Classes, System.Variants,
  FMX.Types, FMX.Controls, FMX.Forms, FMX.Graphics, FMX.Dialogs, FMX.Objects,
  FMX.Edit, FMX.EditBox, FMX.NumberBox, FMX.Controls.Presentation, FMX.StdCtrls;

type
  TBioScreen = class(TForm)
    TitleLabel: TLabel;
    AgeLabel: TLabel;
    NameLabel: TLabel;
    AgeNumberBox: TNumberBox;
    NameEdit: TEdit;
    Image1: TImage;
    StartButton: TButton;
    StyleBook1: TStyleBook;
    procedure StartButtonClick(Sender: TObject);
  private
    { Private declarations }
  public
    { Public declarations }
  end;

var
  BioScreen: TBioScreen;

implementation

{$R *.fmx}

procedure TBioScreen.StartButtonClick(Sender: TObject);
begin
//
end;

end.
